tests/test_gru.py

```diff
@@ -320,6 +368,38 @@ def test_dynamic_bigru_output_consumed_only(self):
         output_names_with_port = ["output:0"]
         self.run_test_case(feed_dict, input_names_with_port, output_names_with_port, rtol=1e-3)
 
+    def test_dynamic_bigru_state_consumed_only(self):
+        units = 5
+        batch_size = 1
+        x_val = np.array([[1., 1.], [2., 2.], [3., 3.]], dtype=np.float32)
+        x_val = np.stack([x_val] * batch_size)
+
+        x = tf.placeholder(tf.float32, x_val.shape, name="input_1")
+        initializer = init_ops.constant_initializer(0.5)
+
+        gru_list = []
```

Comment:
gru_list seems not used?
so are the others.

Reply:
done.